rellic/AST/Z3ConvVisitor.cpp

```diff
@@ -485,7 +486,9 @@ bool Z3ConvVisitor::VisitParenExpr(clang::ParenExpr *parens) {
     // Parens may affect semantics of C expressions
     case Z3_OP_UNINTERPRETED: {
       auto sort = z_sub.get_sort();
+      // DLOG(INFO) << "SATAN 1";
```

Comment:
This looks like it is left over from a frustrating debugging session :).

Reply:
Done.